roottest/root/treeformula/retobj/runretobjTest.C

```diff
@@ -7,8 +7,8 @@ gSystem->Load("libTreePlayer");
 gROOT->ProcessLine(".L helper.C");
 #endif
 
-gSystem->Load("libEvent");
-TFile *Event = TFile::Open("Event.new.split0.root");
+gSystem->Load("libTreeFormulaRetobjGeneration");
+TFile *Event = TFile::Open("Event.root");
```

Comment:
It would be better to avoid this generic file name.

See the discussion in the original PR:
https://github.com/root-project/roottest/pull/773#discussion_r2020254005